yarn-project/aztec-nr/aztec/src/abi.nr

```diff
@@ -29,12 +30,21 @@ struct PublicGlobalVariables {
     version: Field,
     block_number: Field,
     timestamp: Field,
+    coinbase: EthAddress,
+    fee_recipient: AztecAddress,
 }
 // docs:end:public-global-variables
 
-impl Serialize<4> for PublicGlobalVariables {
-    fn serialize(self) -> [Field; 4] {
-        [self.chain_id, self.version, self.block_number, self.timestamp]
+impl Serialize<6> for PublicGlobalVariables {
```

Comment:
I think this implementation is already in the types in `global_variables.nr`, should we not just be importing that so we don't end up repeating.

If not removing it, can you use the constant value instead of a hardcoded number in here?

Reply:
Makes sense. I will try removing it in the [PR up the stack](https://github.com/AztecProtocol/aztec-packages/pull/4473).